scrunch/expressions.py

```diff
@@ -526,7 +527,8 @@ def category_ids(var_id, var_value, variables=variables):
         if len(subitems) == 2:
             if 'value' in subitems[1] and 'variable' in subitems[0]:
                 var_url = subitems[0]['variable']
-                if ds.variables.index[var_url]['type'] == 'multiple_response':
+                var_index = ds.variables.index
+                if var_url in var_index and var_index[var_url]['type'] == 'multiple_response':
                     return adapt_multiple_response(ds, var_url, subitems[1]['value'])
```

Comment:
Note that your return value from `adapt_multiple_response` contains a variable url https://github.com/Crunch-io/scrunch/blob/1c92a46cbd713797c24f9cda85def7c78afea6e3/scrunch/expressions.py#L462 this probably means that you will use that to make another request to that URL to get that variable's attributes yet again, when you already had the variable shoji entity a few lines above. If you return the shoji entity, whoever receives that value can just read from that.